src/libostree/ostree-repo-pull.c

```diff
@@ -117,12 +117,6 @@ typedef struct {
 typedef struct {
   OtPullData  *pull_data;
   GVariant    *object;
-  gboolean     is_detached_meta;
-
-  /* Only relevant when is_detached_meta is TRUE.  Controls
-   * whether to fetch the primary object after fetching its
-   * detached metadata (no need if it's already stored). */
-  gboolean     object_is_stored;
```

Comment:
By deleting this, it looks like we always redownload the commit object too now?  That's not the end of the world, but it does mean another HTTP request by default for "unchanged" state.  Can we still keep this?  Maybe call it `have_attached_object`?